src/clr-angular/forms/styles/_containers.clarity.scss

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('forms.containers') {
+    .clr-form-control {
+        margin-top: $clr-forms-baseline * 4;
+        display: flex;
+        flex-direction: column;
+
+        &.clr-form-control-disabled {
+            opacity: $clr-forms-disabled-opacity;
+        }
+
+        &.clr-form-control-multi {
+            flex-direction: row;
+            flex-wrap: wrap;
+
+            .clr-control-label {
+                width: 100%;
+            }
+        }
+    }
+
+    .clr-control-label {
+        display: block;
+        color: $clr-forms-label-color;
+        @include clr-getTypePropertiesForDomElement(form_label, (font-size));
+        font-weight: 600;
+        line-height: $clr-forms-baseline * 3;
+    }
+
+    .clr-control-container {
+        display: inline-block;
+
+        .clr-subtext {
```

Comment:
Here too. Do we need to target `.clr-subtext`, `.clr-validate-icon` and `.clr-error` inside of `.clr-control-container` only? Or are these styles common across forms. If they are common, we can remove the nesting.

Reply:
Unnested. They are common and unless we used this selector elsewhere it should be fine.